### DISCUSSION TITLE
Dates missing from files view for repos

### DISCUSSION THREAD

**Discussion by omri374** [community/community#43144] [Mobile]
One of the first things I look at for repos is the last updated date. That is the best way to figure out if a project is actively maintained. However, on the app, last modified dates are not shown which makes it difficult to interact with new repos.

**Comment by Professormalik** [comment]
Be patient

**Comment by jfullstackdev** [comment]
as of now, you can have the workaround of using the browser of your mobile, so at least you are still using your phone to use GitHub
